tensorflow/python/autograph/pyct/static_analysis/liveness.py

```diff
@@ -65,10 +65,6 @@ def visit_node(self, node):
       reaching_functions = anno.getanno(
           node.ast_node, anno.Static.DEFINED_FNS_IN)
       for fn_ast_node in reaching_functions:
```

Comment:
If I did a thorough job adding tests, you might need to update liveness_test.py as well.

Reply:
Done